Coding Standards and Procedures

```diff
@@ -0,0 +1,58 @@
+## Standards:
+- Code must be clearly commented.
+  - Contributed code must have a header block including:
+      - general overview in each working file (summary).
+      - Commit history must be appropriate.
+      - Previously existing code is NOT excused from the above requirement.  (*If you change a module, COMMENT IT!)
+  - Comments for each logical block of code are required.  
+      - These comments should explain what that logical block does, default values, values designated for practice, competition, etc. 
+  - Any code that uses UNUSUAL structures or approaches must be clearly explained.
+  - It is recommended that developers follow the literate programming approach: https://csli.sites.stanford.edu/publications/coding-languages/literate-programming.
+- Comments should adhere to the 9 principles outlined in this article: https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/
+  1. Comments should not duplicate the code.
+  2. Good comments do not excuse unclear code.
+  3. If you can’t write a clear comment, there may be a problem with the code.
+  4. Comments should dispel confusion, not cause it.
+  5. Explain UNUSUAL or NONSTANDARD code in comments.
+  6. Provide LINKS to the original source of copied code (if any).
+  7. Include LINKS to external references where they will be most helpful (if any).
+  8. Add comments when fixing bugs.
+  9. Use comments to mark incomplete implementations and/or anything that's need to be done (if any).
```

Comment:
Should this state that the comment should start with "TODO: "?  That way everyone hopefully marks them the same, and so that VS.Code/Visual Studio/<whatever IDE> can readily find them.

For example:
`// TODO: Add a spin flourish to the end of the autonomous routine.`